### DISCUSSION TITLE
New Site Go-Live

### DISCUSSION THREAD

**Discussion by dhambleton** [code4recovery/12-step-meeting-list#1431] [Q&A] [answered]
Hi Community,

I recently went live with a new regional AA website. Throughout development, I had it on a subdomain. When I went live, I switched it from "newsite2024.lancasteraa.org" to just "lancasteraa.org". What is the best way to update the URLs in the public feed? The data source is littered with URLs that still include the subdomain as you can see in the link below. Thank you in advance for any insights!

[Live Website](https://lancasteraa.org)
[Data Source](https://lancasteraa.org/wp-admin/admin-ajax.php?action=meetings)

![image](https://github.com/code4recovery/12-step-meeting-list/assets/160080905/39162cff-69dc-483c-93b2-e6d303ba8296)

**Comment by dhambleton** [comment, marked as answer]
It occurred to me that the file should be updated every time a meeting is updated. I added a meeting, cleared the cache for good measure, and all the URLs in the data source were updated to no longer have the subdomain in them.

**Reply by joshreisner** [reply]
that's right, adding or editing a meeting should refresh any outdated URLs in your feed
